recipes-wpe/wpeframework/wpeframework-plugins_git.bbappend

```diff
@@ -1,15 +1,13 @@
-
-WPE_COMPOSITOR = "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'compositor_wayland', 'compositor_rpi', d)}"
-
 PACKAGECONFIG_append = " ${@bb.utils.contains('DISTRO_FEATURES', 'compositor', '${WPE_COMPOSITOR}', '', d)} \
-                        displayinfo playerinfo"
-
+                        dialserver dial_youtube displayinfo playerinfo power"
 # snapshot implemented with userland support, not applicable on vc4graphics
 PACKAGECONFIG_append = " ${@bb.utils.contains('MACHINE_FEATURES', 'vc4graphics', '', 'snapshot', d)}"
 
-WPE_PLAYERINFO_DEP     = "gstreamer1.0"
-WPE_SNAPSHOT_DEP       = "${@bb.utils.contains('MACHINE_FEATURES', 'vc4graphics', '', 'userland', d)}"
-WPE_WIFICONTROL_DEP    = "linux-firmware-bcm43430"
+# Additional config added for testing
+PACKAGECONFIG_append = " filetransfer inputswitch languageadministrator processmonitor resourcemonitor svalbard systemcommands webproxy webshell"
```

Comment:
Could you please move PACKAGECONFIG to the .conf file? Maybe we can have a _test.conf that can be used with -R option during the build

Reply:
done

Reply:
done